test/e2e/auth/service_accounts.go

```diff
@@ -417,6 +418,119 @@ var _ = SIGDescribe("ServiceAccounts", func() {
 		}
 	})
 
+	/*
+	   Testname: Projected service account token file ownership and permission.
+	   Description: Ensure that Projected Service Account Token is mounted with
+	               correct file ownership and permissino mounted. We test the
+	               following scenarios here.
+	   1. RunAsUser is set,
+	   2. FsGroup is set,
```

Comment:
Can we also have a test case where RunAsUser + FsGroup is set?

Reply:
done